src/CodeGen_D3D12Compute_Dev.cpp

```diff
@@ -1373,8 +1373,8 @@ std::string CodeGen_D3D12Compute_Dev::print_gpu_name(const std::string &name) {
 
 }  // namespace
 
-CodeGen_GPU_Dev *new_CodeGen_D3D12Compute_Dev(const Target &target) {
-    return new CodeGen_D3D12Compute_Dev(target);
+std::unique_ptr<CodeGen_GPU_Dev> new_CodeGen_D3D12Compute_Dev(const Target &target) {
```

Comment:
This appears to build OK everywhere, which is surprising, but maybe we are effectively already using C++14 via the LLVM settings (at least in src/)? In that case, sure